betty/wiki/populator.py

```diff
@@ -244,7 +241,15 @@ async def _image_file_reference(
                     path=image.path,
                     media_type=image.media_type,
                     links=links,
-                    copyright_notice=self._copyright_notice,
+                    copyright_notice=(
+                        await WikimediaCommonsCopyright.new_for_app(
```

Comment:
These factory methods exist so that code can instantiate classes without knowing what arguments their `__init__()` methods require, usually because the class name is not hardcoded. In this case we know the class name, so we can instantiate it without having to call `.new_for_app()`.